src/utils/index.js

```diff
@@ -0,0 +1,19 @@
+export const getRotation = (lastNumberPlate) => {
+  switch (lastNumberPlate.toString()) {
+    case '1':
+    case '2':
+      return 'Segunda-feira';
+    case '3':
+    case '4':
+      return 'Terça-feira';
+    case '5':
+    case '6':
+      return 'Quarta-feira';
+    case '7':
+    case '8':
+      return 'Quinta-feira';
+    case '9':
+    case '0':
+      return 'Sexta-feira';
+  }
```

Comment:
Vamos trocar esse switch case por um objeto que a comparação é muito mais rápida
```suggestion
  const rotation = {
     '1':  'Segunda-feira',
     '2':  'Segunda-feira',
     '3':  'Terça-feira',
     '4':  'Terça-feira',
     '5':  'Quarta-feira',
     '6':  'Quarta-feira',
     '7':  'Quinta-feira',
     '8':  'Quinta-feira',
     '9':  'Sexta-feira',
     '0':  'Sexta-feira',
    }
    
    return rotation[lastNumberPlate.toString()]
```

Reply:
boa, não sabia desta diferença de performance, vou deixar as keys sendo numeros e tirar o  .toString(), pois este metodo faz um match do tipo == e não ===, então não faz tem necessidade de nenhuma converção, neste caso para string, alguns style guides (airbnb e standard) até acusam erro se deixar as keys sendo strings.

Reply:
blz pode ser maninho